webapp/graphite/tags/base.py

```diff
@@ -38,15 +37,29 @@ def find_series(self, tags, timer=None, requestContext=None):
 
     Matching paths are returned as a list of strings.
     """
-    timer.set_name('%s.%s.find_series' % (self.__module__, self.__class__.__name__))
+    start_time = time.time()
+    log_msg = 'completed in'
 
-    cacheKey = 'TagDB.find_series:' + ':'.join(sorted(tags))
-    result = cache.get(cacheKey)
-    if result is not None:
-      timer.set_msg('completed (cached) in')
-    else:
-      result = self._find_series(tags, requestContext)
-      cache.set(cacheKey, result, settings.TAGDB_CACHE_DURATION)
+    try:
+      cacheKey = 'TagDB.find_series:' + ':'.join(sorted(tags))
```

Comment:
would be nice if the cache key generation would be more configurable so additional differentiating factors can be added via configs